app/helpers/users_helper.rb

```diff
@@ -1,2 +1,8 @@
 module UsersHelper
+  def gravatar_for(user, options = {size: 80})
```

Comment:
dùng `(user, size: 80)` cho gọn

Reply:
Số 80 cho vào setting
E tìm hiểu gem config để quản lý các setting của rails application
https://github.com/rubyconfig/config

Reply:
Bước 1: Install Config
- Add `gem 'config'` into Gemfile
```bash
$ bundle install
$ rails g config:install
```
=> Auto Generate
```
config/settings.yml
config/settings.local.yml
config/settings/development.yml
config/settings/production.yml
config/settings/test.yml
```

Bước 2: Add setting for `gravatar_size` in **/config/setting.yml**
```ruby
defaults:
gravatar_size: 80
```
Check lại trong **rails bash**
```bash
$ rails console
3.0.0 :010 > Settings.defaults.gravatar_size
 => 80 
```

Bước 3. Áp dụng Setting.defaults.gravatar_size vào hàm `gravatar_for` trong UsersHelper
```ruby
module  UsersHelper

def  gravatar_for(user)

size = Settings.defaults.gravatar_size

gravatar_id = Digest::MD5.hexdigest(user.email.downcase)

gravatar_url = "https://secure.gravatar.com/avatar/#{gravatar_id}?s=#{size}"

image_tag(gravatar_url, alt:  user.name, class:  'gravatar')

end

end
```